landing-page/src/About.jsx

```diff
@@ -0,0 +1,35 @@
+import Button from './Button.jsx';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import aboutFoto from "./content/aboutFoto.jpg";
+import { faLinkedin, faInstagram } from '@fortawesome/free-brands-svg-icons';
+import { faAt, faGraduationCap, faLocationDot, faPaperPlane} from '@fortawesome/free-solid-svg-icons';
+
+function About(){
+    return (
+        <div className="about">
+        <img className="aboutFoto" src={aboutFoto} alt="author foto"/>
+        <div className="aboutInfo">
+            <h2>More Info About Me</h2>
+            <p>I am an enthusiastic and detail-oriented beginner Front-end Developer,
+            seeking an entry position in a company, to use and improve my skills in 
+            commercial project development. While studying I'm trying to have a deep
+            understanding of every new technology and am always highly motivated to 
+            solve the unfamiliar complex tasks</p>
+            <div className="aboutDetails">
+                <p><FontAwesomeIcon className="aboutDetailsIcon" icon={faLinkedin}/><a href="https://linkedin.com/in/trandobulo/">Linkedin</a></p>
+                <p><FontAwesomeIcon className="aboutDetailsIcon" icon={faAt}/><a href="mailto: trandobulo@gmail.com">trandobulo@gmail.com</a></p>
+                <p><FontAwesomeIcon className="aboutDetailsIcon" icon={faGraduationCap}/><a href="https://onmu.odessa.ua">ONMU</a></p>
+                <p><FontAwesomeIcon className="aboutDetailsIcon" icon={faInstagram}/><a href="instagram.com//trand.oleksandr">Instagram</a></p>
+                <p><FontAwesomeIcon className="aboutDetailsIcon" icon={faPaperPlane}/><a href="https://t.me/trand_alexandr">+380938190948</a></p>
+                <p><FontAwesomeIcon className="aboutDetailsIcon" icon={faLocationDot}/>Odessa, Ukraine</p>

```

Comment:
Please do it in a declarative way. Something like this:

```
[{ icon: ... , link: ... , text: ... }].map(
    (x) => <p><FontAwesomeIcon icon={x.icon}><a href={x.link}>{x.text}</a><p>
)
```

Reply:
You mean like that ?: 

`
function AboutDetails(){
    return (
    [{icon: faLinkedin, link: "https://linkedin.com/in/trandobulo/", text:"Linkedin"},
     {icon: faAt, link: "mailto: trandobulo@gmail.com", text:"trandobulo@gmail.com"},
     {icon: faGraduationCap, link: "https://onmu.odessa.ua", text:"ONMU"},
     {icon: faInstagram, link: "instagram.com//trand.oleksandr", text:"Instagram"},
     {icon: faPaperPlane, link: "https://t.me/trand_alexandr", text:"+380938190948"},
     {icon: faLocationDot, text:"Odessa, Ukraine"}
     
    ].map((elem) => <p><FontAwesomeIcon className="aboutDetailsIcon" icon={elem.icon}/><a href={elem.link}>{elem.text}</a></p>)
)
}`

Reply:
you mean like that ? 

```suggestion
                
function AboutDetails(){
    return (
    [{icon: faLinkedin, link: "https://linkedin.com/in/trandobulo/", text:"Linkedin"},
     {icon: faAt, link: "mailto: trandobulo@gmail.com", text:"trandobulo@gmail.com"},
     {icon: faGraduationCap, link: "https://onmu.odessa.ua", text:"ONMU"},
     {icon: faInstagram, link: "instagram.com//trand.oleksandr", text:"Instagram"},
     {icon: faPaperPlane, link: "https://t.me/trand_alexandr", text:"+380938190948"},
     {icon: faLocationDot, text:"Odessa, Ukraine"}
     
    ].map((elem) => <p><FontAwesomeIcon className="aboutDetailsIcon" icon={elem.icon}/><a href={elem.link}>{elem.text}</a></p>)
)
}
```

Reply:
Yes, something like that. In case of creating the separate render function `AboutDetails` you can add the wrapper div as well:

```suggestion
const AboutDetails = () => {
 const items =  [{icon: faLinkedin, link: "https://linkedin.com/in/trandobulo/", text:"Linkedin"},
     {icon: faAt, link: "mailto: trandobulo@gmail.com", text:"trandobulo@gmail.com"},
     {icon: faGraduationCap, link: "https://onmu.odessa.ua", text:"ONMU"},
     {icon: faInstagram, link: "instagram.com//trand.oleksandr", text:"Instagram"},
     {icon: faPaperPlane, link: "https://t.me/trand_alexandr", text:"+380938190948"},
     {icon: faLocationDot, text:"Odessa, Ukraine"}]

const AboutItem = (props) => (<p><FontAwesomeIcon className="aboutDetailsIcon" icon={props.icon}/><a href={props.link}>{props.text}</a></p>)
}

return (<div>{items.map(x => <AboutItem item={x})</div>)
```